core/cp4d_authenticator.go

```diff
@@ -32,35 +32,36 @@ const (
 	PRE_AUTH_PATH = "/v1/preauth/validateAuth"
 )
 
-// CloudPakForDataAuthenticator : This authenticator will automatically fetch an access token for the
-// user-specified username and password.  Outbound REST requests invoked by the BaseService are then authenticated
-// by adding a Bearer-type Authorization header containing the access token.
+// CloudPakForDataAuthenticator utilizes a username and password pair to
+// obtain a suitable bearer token, and adds it requests.
+//
+// The bearer token will be sent as an Authorization header in the form:
+//
+// 		Authorization: Bearer <bearer-token>
 type CloudPakForDataAuthenticator struct {
-	// [Required] The URL representing the token server's endpoing.
```

Comment:
I think it's a mistake to remove the "required"/"optional" information from these comments.   I specifically put it there to try to help the user understand what is and is not required when constructing an instance of the authenticator.

Reply:
👍 Ok, Lets add them back. I didn't see any standard for them, but I can see how it would be helpful.